src/style.css

```diff
@@ -2,0 +1,30 @@
+.body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-image: url('./images/star.svg');
+  margin-right: 4px;
+}
```

Comment:
You're using a class selector `.stars__star` to style your stars. This is a good practice. However, it's better to avoid using tag names for styling. You should use `.classes` and if needed with `:pseudo-classes`, `pseudo-elements` and `[attributes]`.